sigal/image.py

```diff
@@ -160,6 +161,20 @@ def process_image(filepath, outpath, settings):
     return Status.SUCCESS
 
 
+def get_size(file_path):
+    """Return image size (width and height)."""
+    logger = logging.getLogger(__name__)
+    try:
+        im = PILImage.open(file_path)
+    except:
+        logger.error("Failed to open %s", file_path)
```

Comment:
With this try/except, the try/except in the properties is useless and can be removed. Also the error message could be more specific, like above (`Could not read size ...`), and specifying the list of exceptions (`IOError, IndexError, TypeError, AttributeError` should be fine) is better practise.